lib/builder/webpack/client.js

```diff
@@ -99,31 +99,36 @@ export default class WebpackClientConfig extends WebpackBaseConfig {
   extendConfig() {
     const config = super.extendConfig(...arguments)
 
-    if (!this.options.dev && !config.optimization.minimizer) {
+    // Add minimizer plugins
+    if (config.optimization.minimize && config.optimization.minimizer === undefined) {
       config.optimization.minimizer = []
 
       // https://github.com/webpack-contrib/terser-webpack-plugin
-      const terserJsPlugin = new TerserWebpackPlugin({
-        parallel: true,
-        cache: this.options.build.cache,
-        sourceMap: config.devtool && /source-?map/.test(config.devtool),
-        extractComments: {
-          filename: 'LICENSES'
-        },
-        terserOptions: {
-          output: {
-            comments: /^\**!|@preserve|@license|@cc_on/
-          }
-        }
-      })
-      config.optimization.minimizer.push(terserJsPlugin)
+      if (this.options.build.terser) {
```

Comment:
So if minifier configed, teaser config is useless. Do we still need this check?

Reply:
How about enable teaser if minifie enabled and no specified minifier.
Then this check can be removed

Reply:
It is to disable terser only. `optimize.minify` is a webpack option indeed.

https://webpack.js.org/configuration/optimization